src/grt/include/grt/GlobalRouter.h

```diff
@@ -195,7 +195,7 @@ class GlobalRouter : public ant::GlobalRouteSource
   bool isCoveringPin(Net* net, GSegment& segment);
   std::vector<Net*> initFastRoute(int min_routing_layer, int max_routing_layer);
   void initFastRouteIncr(std::vector<Net*>& nets);
-  void estimateRC();
+  void estimateRC(const char* path = "");
```

Comment:
Can we change this to be std::ostream instead of a path? It makes it way more versatile, and easier to test.